configure.ac

```diff
@@ -57,6 +57,13 @@ PKG_CHECK_MODULES([LO], [liblo >= 0.23], [],
 PKG_CHECK_MODULES([TINYXML2], [tinyxml2 >= 6], [],
 	AC_MSG_ERROR([tinyxml2 library >= 6.0.0 not found]))
 
+# check for sensor constants
+if test `uname -s` = Darwin; then
+# make sure that the SDL2 headers from Homebrew are found on Darwin arm64
+CPPFLAGS="-I/opt/homebrew/include ${CPPFLAGS}"
```

Comment:
Is this needed? I would think pkg-config provides this when it is called by PKG_CHECK_MODULES([SDL]... ie.
~~~
% pkg-config sdl2 --cflags
-D_THREAD_SAFE -I/opt/homebrew/include -I/opt/homebrew/include/SDL2
~~~

Reply:
Ah never mind, adding the CFLAGS is needed but we can use the SDL_CFLAGS provided by PKG_CHECK_MODULES: `CPPFLAGS="$CPPFLAGS $SDL_CFLAGS"`

Reply:
Ok, I can still add this to #20.

Reply:
Done.